_shared/project/tox.ini

```diff
@@ -50,7 +56,7 @@ passenv =
     {% endif %}
 deps =
 {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
-    pip-tools
+    -r requirements/{env:TOX_ENV_NAME}.txt
```

Comment:
This is part of changing from calling `pip-sync` directly to using `pip-sync-faster`. It's documented here: https://stackoverflow.com/a/73503093/1175266